### ISSUE TITLE
Reduce maximum latency for concurrent requests to all group pages

### ISSUE THREAD

**Issue by giorgiosironi** [sciety/sciety#1457]
https://docs.google.com/spreadsheets/d/1WgJltkh987AwNjV51huJoQAElSxEtKCEYE4GS7GS264/edit#gid=209050637

**Comment by giorgiosironi** [comment]
We encountered this problem in the infrastructure build:
```
Warning: Deprecated Resource
Error: Your query returned no results. Please change your search criteria and try again.

  on .terraform/modules/kubernetes_cluster.eks/data.tf line 36, in data "aws_ami" "eks_worker_windows":
  36: data "aws_ami" "eks_worker_windows" ***
```

That file suggests:
```
  # Windows nodes are available from k8s 1.14. If cluster version is less than 1.14, fix ami filter to some constant to not fail on 'terraform plan'.
  worker_ami_name_filter_windows = (var.worker_ami_name_filter_windows != "" ?
    var.worker_ami_name_filter_windows : "Windows_Server-2019-English-Core-EKS_Optimized-${tonumber(var.cluster_version) >= 1.14 ? var.cluster_version : 1.14}-*"
  )
```
might be related to us using a very old version of the module (`8.2.0`).

**Comment by giorgiosironi** [comment]
Saving a query for later:
https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:Windows_Server-2019-English-Core-EKS_Optimized
